app/views/course/dashboards/show.html.erb

```diff
@@ -1,21 +1,16 @@
-<h1 class="mb-3">Welcome Back!</h1>
-<div class="mb-4">
-  <div class="mb-3">
-    <p>You've completed <%= number_to_percentage(current_user.course_status.completion_rate * 100, precision: 0) %> of the sections in the course.</p>
-    <%= render "status/progress", status: current_user.course_status %>
+<h1 class="mb-4 text-center">Welcome Back!</h1>
+<div class="row">
+  <div class="col-lg-6 col-12">
+    <div class="card p-3 text-center">
+      <%= render current_user.course_status.performance %>
+    </div>
+  </div>
+  <div class="col-lg-6 col-12">
+    <div class="card p-3 text-center mb-4">
+      "<%= QuoteFetcher.execute.sample["text"] %>"<br><hr>-<%= QuoteFetcher.execute.sample["author"] %>
```

Comment:
As mentioned before. as long as we cache this, I'm fine with the call. A side effect of not caching and the way you're calling it in here, is that the text and author are getting randomized.
There are 2 API calls, and 2 different sample calls on 2 different sets of quotes.

So:
![image](https://user-images.githubusercontent.com/1820441/113651605-e8622780-9657-11eb-8724-3f96ac3a9e63.png)

You'll end up with: :point_up: 
> "To be beautiful  means to be yourself. You do not need to be accepted by others. You need to accept yourself.",
> Carl Jung

Also... we need to be careful with this API. :joy: 
![image](https://user-images.githubusercontent.com/1820441/113651382-8275a000-9657-11eb-9d50-9a7ff657a454.png)